lib/node_modules/@stdlib/blas/ext/base/sindex-of/README.md

```diff
@@ -0,0 +1,196 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2025 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# sindexOf
+
+> Return the index of a specified search element in a single-precision floating-point strided array.
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- Package usage documentation. -->
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var sindexOf = require( '@stdlib/blas/ext/base/sindex-of' );
+```
+
+#### sindexOf( N, searchElement, x, strideX )
+
+Returns the index of a specified search element in a single-precision floating-point strided array.
+
+```javascript
+var Float32Array = require( '@stdlib/array/float32' );
+
+var x = new Float32Array( [ -2.0, 1.0, 3.0, -5.0, 4.0, 0.0, -1.0, 3.0 ] );
+
+var idx = sindexOf( x.length, 3.0, x, 1 );
+// returns 2
+```
+
+The function has the following parameters:
+
+-   **N**: number of indexed elements.
+-   **searchElement**: element to search the index of.
+-   **x**: input [`Float32Array`][@stdlib/array/float32].
+-   **strideX**: index increment.
+
+If the function is unable to find an element which equals a provided search element, the function returns `-1`.
+
+```javascript
+var Float32Array = require( '@stdlib/array/float32' );
+
+var x = new Float32Array( [ -2.0, 1.0, 3.0, -5.0, 4.0, 0.0, -1.0, 3.0 ] );
+
+var idx = sindexOf( x.length, 8.0, x, 1 );
+// returns -1
+```
+
+The `N` and stride parameters determine which elements in the strided array are accessed at runtime. For example, to search every other element:
+
+```javascript
+var Float32Array = require( '@stdlib/array/float32' );
+
+var x = new Float32Array( [ -2.0, 1.0, 3.0, -5.0, 4.0, 0.0, -1.0, 3.0 ] );
+
+var idx = sindexOf( 4, -1.0, x, 2 );
+// returns 6
```

Comment:
This is incorrect. `N` specifies the view length. Here, that "view" is four elements. `6` is the index in the underlying buffer. That is not what we care about in this case.

Reply:
See also [`blas/base/isamax`](https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/blas/base/isamax).